### PR TITLE
Add docs on managing users, teams, repos.

### PR DESCRIPTION
**Pull request by HumairAK**
Also snuck in some minor clean up in the github-config.yaml file.

### CODE REVIEW

**Review by harshad16 — COMMENTED** [review]
lgtm
very neat 
thanks :100:

File: docs/add_gh_member_and_access.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+All Operate First GitHub members and teams are stored declaratively in the `github-config.yaml` file. Once this file is updated, our
+[Peribolos][peri] deployment will bring these changes live. Thus, to add new members or teams and manage team access,
+one needs to simply update this file, make a PR, and once merged, the changes will take effect.
+
+## Table of contents
+
+- [Become a Github member](#become-a-github-member)
+- [Get repo access](#get-repo-access)
+- [Create a GitHub team](#Create-a GitHub-team)
```
Comment (4n4nd) [inline docs/add_gh_member_and_access.md:11]: link doesn't seem to work

File: docs/add_gh_member_and_access.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+- Being able to be assigned GitHub `Issues` in any Operate First repo
+- Create Discussions
+- CI jobs run automatically on PRs without the need for `/ok-to-test` by another member
+- Gain the ability to use certain chat ops commands on issues and pull requests
+- Gain ability to join Operate First GitHub teams and gain elevated access to repos
+
+To become a GitHub member follow these steps:
+
+1. Fork/Clone the repo
```
Comment (4n4nd) [inline docs/add_gh_member_and_access.md:24]: which repo?

**Review by 4n4nd — APPROVED** [review]
/lgtm

**Review by 4n4nd — APPROVED** [review]
/lgtm

**Pull request merged**